.github/workflows/sandbox-rebuild.yml

```
@@ -0,0 +1,28 @@
+  AWS_REGION: ${{ vars.AWS_REGION }}
+  BRANCH_NAME: ${{ github.head_ref || github.ref_name }} 
+
+permissions:
+  id-token: write # This is required for requesting the JWT
+  contents: read # This is required for actions/checkout
+jobs:
+  AssumeRoleAndStartExecution:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

actionlint (1.7.3)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)